templates/lobbys/lobby.html

```diff
@@ -177,10 +211,45 @@ <h2 v-else>Lobby [[lobbyInfo.lobby_id]]: </h2>
         <span id="custom-tooltip" ref="shareTooltip">Copied invite to clipboard!</span>
     </div>
 
+    
 
     <template v-if="lobbyInfo.user.owner">
+
         <hr>
-        <h4>Manage: Add a prompt</h4>
+
+        <h3>Lobby settings</h4>
+
+        <details>
+            <summary>List of players</summary>
+            <div class="card">
+                <div class="card-body">
+                    <table class="table table-hover">
+                        <thead>
+                            <th scope="col">Display name</th>
+                            <th scope="col">Host</th>
+                            <th scope="col">Full user</th>
+                            <th scope="col"></th>
+                        </thead>
+                        <tbody>
+                            <tr v-for="user in users" v-cloak>
+                                <td>[[user.username]]</td>
+                                <td><i v-if="user.owner" class="bi bi-check"></i></td>
+                                <td><i class="bi bi-check"></i></td>
+                                <td><button v-if="!user.owner" class="btn btn-link btn-sm" @click="makeHost(user.user_id, user.username)"><small>Make host</small></button></td>
```

Comment:
See above comment regarding "Make host" text

But in addition:

(Personal preference) 
I think this should probably be a dropdown menu with options per user (for now, this can be the only thing in the dropdown).

1. It would be cleaner UI
2. We probably want to support more user management options in the future, namely kicking or banning users from the lobby, and this would be a clean and extensible way to support those additional options per user.